misc/remove_libraries_for_other_platforms.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+OS=$(uname)
+
+panic() {
+	printf "%s\n" "$1"
+	exit 1
+}
+
+remove_windows_libraries() {
+	find . -type f -name '*.dll' | xargs rm -f 
+	find . -type f -name '*.lib' | xargs rm -f
+	find . -type d -name 'windows' | xargs rm -rf
+}
+
+remove_macos_libraries() {
+	find . -type f -name '*.dylib' | xargs rm -f 
+	find . -type d -name '*macos*' | xargs rm -rf
+}
+
+remove_linux_libraries() {
+	find . -type f -name '*.so' | xargs rm -f 
+	find . -type d -name 'linux' | xargs rm -rf
+}
+
+case $OS in
+	Linux)
```

Comment:
Maybe there could be an `assert_vendor` sub routine that verifies you're calling it from `vendor`, using `basename $(pwd)`?

Just so that you're not accidentally nuking anything else.

Reply:
Just added in new commit

Reply:
Great, but you'd probably want to run that on Darwin and *BSD as well?

Reply:
Done